doc-content/enterprise-only/processes/interacting-with-processes-releasing-tasks-in-bulk-proc.adoc

```diff
@@ -0,0 +1,12 @@
+[id='interacting-with-processes-releasing-tasks-in-bulk-proc']
+= Releasing tasks in bulk
+
+After tasks are created in {CENTRAL}, you can release your tasks in bulk for others to claim.
```

Comment:
@kaldesai it's not enough that task are created, you need they are assigned to you. I would remove the first part: After tasks are created in {CENTRAL}. Maybe introduce 'your owned tasks' here would be clearer that you are the owner

Reply:
Hey @nmirasch, 
wdyt?
I will just add the following sentence..
"_You can release your owned tasks in bulk for others to claim._"

Reply:
looks good!